instrumentation/logback/logback-appender-1.0/library/src/main/java/io/opentelemetry/instrumentation/logback/appender/v1_0/OpenTelemetryAppender.java

```diff
@@ -62,6 +65,17 @@ public void setCaptureExperimentalAttributes(boolean captureExperimentalAttribut
     this.captureExperimentalAttributes = captureExperimentalAttributes;
   }
 
+  /**
+   * Sets whether the code attributes (file name, class name, method name and line number) should be
+   * set to logs.
```

Comment:
https://logback.qos.ch/manual/layouts.html states `Generating the line number information is not particularly fast. Thus, its use should be avoided unless execution speed is not an issue.` (same for other code attributes). Should we also mention this here?

Reply:
Sure, I have added a [sentence about this](https://github.com/jeanbisutti/opentelemetry-java-instrumentation/blob/d9bbf85f260f4b63809a34f7d02e5311d811de06/instrumentation/logback/logback-appender-1.0/library/src/main/java/io/opentelemetry/instrumentation/logback/appender/v1_0/OpenTelemetryAppender.java#L70).